content/webapp/components/ContentSearchResult/index.tsx

```diff
@@ -62,9 +63,12 @@ const ContentSearchResult: FunctionComponent<Props> = ({
   dates,
   times,
   contributors,
+  positionInList,
 }) => {
   return (
     <Link
+      data-gtm-trigger="search_result"
+      data-gtm-position-in-list={positionInList}
```

Comment:
I didn't see this fire - maybe easier on a call as I am probably testing it wrong!

Reply:
I'm just updating this to distinguish between `all_search_results_click`s and `works_search_result_click`s

Reply:
You should see these differentiated now (you have to have the GA rather than the GTM tag selected)

<img width="426" alt="Screenshot 2025-05-29 at 10 35 22" src="https://github.com/user-attachments/assets/c5f7fc1e-b1c8-450c-a164-801904b43d81" />